setup.py

```diff
@@ -35,11 +35,12 @@
 # IMPORTANT: keep in sync with .github/workflows/build.yml
 #
 # Version of libsecp256k1 to download if none exists in the `libsecp256k1` directory
-UPSTREAM_REF = os.getenv('COINCURVE_UPSTREAM_REF') or 'ddf2b2910eb19032f8dd657c66735115ae24bfba'
+# UPSTREAM_TAG = os.getenv('COINCURVE_UPSTREAM_TAG') or 'v0.3.2'
```

Comment:
```suggestion
```

Reply:
For my understanding, why is the REF preferred to the TAG?

Reply:
tags can be altered to point to different commits but commit hashes can never change